### PR TITLE
Feedback / pre-release notice

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**

### CODE REVIEW

File: src/components/HelpMenu.tsx [hunk 1]
```diff
@@ -100,7 +112,7 @@ const HelpMenu = ({ isMobile, ...rest }: HelpMenuProps) => {
                 onClick={deployment.compliance.manageCookies}
                 icon={<MdOutlineCookie />}
               >
-                <FormattedMessage id="cookies-action" />
+                <FormattedMessage id="help-support" />
```
Comment (microbit-matt-hillsdon) [inline src/components/HelpMenu.tsx:115]: Um no!

Comment (microbit-matt-hillsdon) [reply src/components/HelpMenu.tsx]: f8367537e273b5e3acc7d1417e82d70abdaf6dee

**Comment by microbit-matt-hillsdon** [comment]
I have successfully sent/received a test submission.

File: src/flags.ts [hunk 2]
```diff
@@ -16,9 +16,9 @@ export type Flag =
    */
   | "devtools"
   /**
-   * Flag to add a prototype warning. Enabled for staging site and production stages.
+   * Flag to add a beta warning. Enabled for staging site and production stages.
```
Comment (microbit-grace) [inline src/flags.ts:19]:
```suggestion
   * Flag to add a beta warning. Enabled for review and staging site stages.
```

**Review by microbit-grace — COMMENTED** [review]
Looks good, added a comment suggestion

**Pull request merged**